jennchenn/uwexplore
# Hover lecture times

## Implementation description

- Hovering over the add button on the search cards will make a ghost course appear on the calendar to see the different section times
- Currently, all the courses that are hoverable are also the ones that are on the cal, so this may look a little weird? 
- There's no option  to hover over a specific lecture/tutorial time yet (to see when different sections fit in the cal), but I've made a ticket for this in Linear
- Basically passed props in a chain from App.tsx, someone stop me if this is not good practice



## Steps to test

1. Hover over the add button on the search cards and verify the course sections appear on the calendar



## What should reviewers focus on?

-
